test_programs/compile_success_empty/serialize/src/main.nr

```diff
@@ -7,7 +7,7 @@ trait Serialize {
 
 impl<A, B, let AS: u32, let BS: u32> Serialize for (A, B) where A: Serialize<Size = AS>, B: Serialize<Size = BS> {
     // let Size = <A as Serialize>::Size + <B as Serialize>::Size;
-    let Size = AS + BS;
+    let Size: u32 = AS + BS;
```

Comment:
Ideally these wouldn't be needed and we'd copy it from the trait definition but we can fix that in a later PR.